helm/hpcc/templates/thor.yaml

```diff
@@ -189,7 +189,9 @@ data:
 
 {{ range $.Values.thor -}}
 {{- if not .disabled -}}
-{{- $commonCtx := dict "root" $ "me" . -}}
+{{- $secretsCategoriesUser := list  "storage" "ecl" "ecl-user" }}
```

Comment:
I'm not sure if these do need to be different, but if they are they should be something like secretsCategoriesManager and secretsCategoriesAgent